### DISCUSSION TITLE
Open Source Data

### DISCUSSION THREAD

**Discussion by clizbe** [TulipaEnergy/TulipaEnergyModel.jl#329] [Ideas]
Let's use this discussion to track all cool open data sources we run into that might be useful later on:
https://openinframap.org/#9.34/52.2069/4.3258
https://corres.windenergy.dtu.dk/

**Comment by clizbe** [comment]
Email from Mopo about Hydro:

Hi all,

It seems like what the Mopo project wants out of hydropower has already been done on some level here:
https://www.sciencedirect.com/science/article/pii/S2589004221009676
https://github.com/ebbekyhl/Future-operation-of-hydropower-in-Europe/tree/v1.0
They use the atlite package by PyPSA( ? ) team:
https://atlite.readthedocs.io/en/latest/introduction.html
This could be a good reference for Mopo, as they seem implement something for dynamic line rating and heating as well.
Some possible improvements:
•	add possibility for multiple upstream basins per hydropower plant
•	replace Volue Insight reference data with some open source
•	improve on JRC hydropower database
•	use MERIT-Hydro or HydroSHEDS v2 ( once it is published ) instead of HydroSHEDS
•	use ERA5-Land instead of ERA5
•	add lake evaporation
•	model where the runoff comes from instead of just taking it from the a reanalysis product 
o	i.e. what would historical precipitation with slightly higher temperatures produce
Best regards,

Tommi Laine
Fortum

**Reply by clizbe** [reply]
Maybe not open-source, but Sebastiaan suggested:
VERBUND (Austria's electricity company)

Pypsa might have data we can use.

**Comment by clizbe** [comment]
https://ens.dk/en/our-services/technology-catalogues
https://energy.ec.europa.eu/data-and-analysis/energy-modelling/eu-reference-scenario-2020_en

**Comment by clizbe** [comment]
Tons of sources here:
https://zenodo.org/search?q=nswph&l=list&p=1&s=10&sort=bestmatch

**Comment by clizbe** [comment]
Not open source, but looks like maybe detailed windpark data:
https://www.thewindpower.net/store_continent_en.php?id_zone=1001

**Comment by clizbe** [comment]
Dunno if it's opensource, but @datejada found this:
https://www.linkedin.com/posts/activity-7140005469414133760-f4XH/?utm_source=share&utm_medium=member_desktop

**Comment by clizbe** [comment]
Recently made available through a TNO project with Liander:
https://www.liander.nl/over-ons/open-data

Unfortunately I think it's all in Dutch....

**Comment by clizbe** [comment]
Not sure if it's opensource, but looks like a good resource for Dutch data:
https://energiedatawijzer.nl/over-de-energiedatawijzer/

**Comment by clizbe** [comment]
[PUDL](https://catalyst.coop/pudl/) is an opensource data provider (mostly U.S. data) that provides their data on multiple platforms and many formats etc.
They seem great! Even if we don't use them for data, we can consider HOW they do data.
Or maybe use their data for an open-source proof/comparison that's replicable, if we ever need that.

**Reply by clizbe** [reply]
And they're opensource so we could see their processing system etc.

**Comment by clizbe** [comment]
OpenStreetMap has a bunch of high quality infrastructure data. For questions talk to Tobias now at OET.
There might be more, but this is at least a reference of a lot of it:
https://github.com/open-energy-transition/Awesome-Electric-Grid-Mapping

**Comment by clizbe** [comment]
Rijkswaterstraat consolidates and maintains a bunch of Dutch data that they're supposedly making open soon.
It's called "CentraalToegangspuntData" and works with SQL.
This could be a **major source** once it's open.

**Comment by clizbe** [comment]
**Other Countries**
[Estonia](https://e-estonia.com/solutions/e-governance/e-services-registries/) 
[France](https://cartagene-cerema-fr.translate.goog/portal/apps/experiencebuilder/experience/?_x_tr_hist=true&_x_tr_hl=nl&_x_tr_pto=wapp&_x_tr_sl=fr&_x_tr_tl=en&id=d1e7a5b177d14c83b3be4f3be6af85cf)

**Comment by clizbe** [comment]
Here's an amalgamation of grid data:
https://www.linkedin.com/posts/maartenstaats_european-capacitymap-remaining-activity-7325427120124436480-jWEE?utm_source=share&utm_medium=member_android&rcm=ACoAABQ_emwBocRk9l7xlEmhfgHl3q02f5EahMU

**Comment by clizbe** [comment]
Liander posted 15 min usage data:
https://www.liander.nl/over-ons/open-data#historische-15-minuten-bedrijfsmetingen

**Comment by clizbe** [comment]
Probably more relevant to Built Env than us, but still cool!
https://gizmodo.com/literally-a-map-showing-all-the-buildings-in-the-world-2000694696

**Comment by ve524** [comment]
But I don't know how to use it.
